### PR TITLE
[release-4.14] OCPBUGS-20472: hosted cluster upgrade failure from 4.13 stable to 4.14

### PR DESCRIPTION
**Pull request by flavio-fernandes**
Add missing volume mount to ovnkube-node pods while in the single zone phase

### CODE REVIEW

**Comment by ricky-rav** [comment]
I prefer this solution to having the ovnkube-config ConfigMap copied in two places :)
Let's try it out.
Thanks, Flavio!

File: bindata/network/ovn-kubernetes/managed/single-zone-interconnect/ovnkube-node.yaml [hunk 1]
```diff
@@ -658,6 +658,10 @@ spec:
           name: host-var-lib-cni-networks-ovn-kubernetes
         - mountPath: /run/openvswitch
           name: run-openvswitch
+        # gives write access to libovsdb logs until moving to multi-zone
```
Comment (flavio-fernandes) [inline bindata/network/ovn-kubernetes/managed/single-zone-interconnect/ovnkube-node.yaml:661]: @tssurya is this comment in line with what you had in mind?

**Review by trozet — APPROVED** [review]
/approve

**Pull request merged**